ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/vm/MessageFrame.java

```diff
@@ -29,10 +29,7 @@
 import tech.pegasys.pantheon.util.uint.UInt256;
 import tech.pegasys.pantheon.util.uint.UInt256Value;
 
-import java.util.Deque;
-import java.util.EnumSet;
-import java.util.HashSet;
-import java.util.Set;
+import java.util.*;
```

Comment:
We have a coding standard around not collapsing imports.

Reply:
This is my IDE, when the number of imports of the same package is higher than a threshold it replaces with *. i need to disable this

Reply:
yeah - i set mine to like 9999 or something :P

Reply:
fixed